modules/virt-deleting-vmis-web.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-manage-vmis.adoc
+
+[id="virt-deleting-vmis-web_{context}"]
+= Deleting a standalone virtual machine instance using the web console
+
+
+.Procedure
+
+. In the {VirtProductName} console, click *Workloads* -> *Virtual Machines* from the side menu.
+. Click the &#8942; button of the standalone virtual machine instance (VMI) that you want to delete and select *Delete Virtual Machine Instance*.
+** Alternatively, click the standalone VMI name. The *Virtual Machine Overview* page displays.
+
+. Click the *Details* tab. The *Virtual Machine Details* page displays.
```

Comment:
This step is not necessary, `Action` button is available in every tab, includes `Overview` tab.

Reply:
Removed the step.